docs/content/docs/workflows__pages.md

```diff
@@ -0,0 +1,58 @@
+# Workflow pages
+
+Each workflow has a set of steps, those steps have pages associated with them. For most cases there are at least two pages: 
+
+- Input, a Simput panel for modifiying or creating the simulation input.
+- Submission, starts the job
+- Monitor, monitors the job
+
+These pages can be built off the components in: 
+
+```sh
+src/workflows/generic/components/steps
+```
+
+## Input
+
+Import the component `generic/components/steps/SimputReact` and pass the following props: 
+
+- `simputType`, the name of the simput type
+- `inputFileKeys`, arguments to simput on the key for the file in the simulation metadata `key: String`, the name of the file `name: String`, and if the input file should be parsed `parse: Boolean`
```

Comment:
Those are not necessary for simput. This list the files that either come in your project and/or simulation page that should be considered as Input and pass along the flow.